### PR TITLE
Principal ideal quotient fp

### PR DESCRIPTION
**Pull request by felixwellen**
Joint work with @MatthiasHu :
Construct finite representaion of quotients by principal ideals.
Depends on #810 -> Draft

### CODE REVIEW

File: Cubical/Algebra/CommAlgebra/Base.agda [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{-# OPTIONS --safe #-}
+{-# OPTIONS --safe --postfix-projections #-}
```
Comment (MatthiasHu) [inline Cubical/Algebra/CommAlgebra/Base.agda:1]: I think it was unintended to commit this?

File: Cubical/Algebra/CommAlgebra/Instances/Initial.agda [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{-# OPTIONS --safe #-}
+{-# OPTIONS --safe --postfix-projections #-}
```
Comment (MatthiasHu) [inline Cubical/Algebra/CommAlgebra/Instances/Initial.agda:1]: (Here also.)

**Comment by felixwellen** [comment]
@MatthiasHu thanks!

**Comment by felixwellen** [comment]
The depency got replaced (by #816 ), and the replacement is merged. So this can be merged as well.

**Comment by felixwellen** [comment]
The PR also contains a fruitless attempt to make things type check faster by using more copatterns. Since this is a rule anyway, I'll leave that as it is. I'll have a look if the PR checks with latest master and merge if yes.

**Pull request merged**